src/chainparamsseeds.h

```diff
@@ -889,6 +889,13 @@ static SeedSpec6 pnSeed6_main[] = {
     {{0xfd,0x87,0xd8,0x7e,0xeb,0x43,0xcf,0x7b,0x5e,0x3a,0x53,0x21,0x5b,0x62,0xe3,0x7a}, 8333}
 };
 
+static SeedSpec6 pnSeed6_seg[] = {
```

Comment:
would be nice if the code has more comments that referred to documentation that explains what some of these numbers are